build.assets/genproto.sh

```diff
@@ -74,7 +74,10 @@ main() {
   [[ $skip_js -eq 0 ]] && echoed buf generate --template=buf-js.gen.yaml \
     --path=proto/prehog/ \
     --path=proto/teleport/lib/teleterm/ \
-    --path=api/proto/teleport/userpreferences/
+    --path=api/proto/teleport/userpreferences/ \
+    --path=api/proto/teleport/accesslist/ \
+    --path=api/proto/teleport/trait/ \
+    --path=api/proto/teleport/header/
```

Comment:
These are dependencies of `accesslist`.

Reply:
I added #35828 which makes sure that the tests will fail if dependencies are not generated properly.